doc_src/cmds/bind.rst

```diff
@@ -87,6 +87,8 @@ The following special input functions are available:
 
 - ``backward-kill-word``, move the word to the left of the cursor to the killring. The "word" here is everything up to punctuation or whitespace.
 
+- ``backward-path-component``, move one path component to the left. A path component is everything likely to belong to a path component, i.e. not any of the following: `/={,}'\":@ |;<>&`, plus newlines and tabs.
```

Comment:
I don't think "to the left" is great here, because it conflates "left" with "backward" (which is not true of all text, although I certainly don't know how RTL text works across all commandlines).

```suggestion
- ``backward-path-component``, move one path component backward. A path component is everything likely to belong to a path component, i.e. not any of the following: `/={,}'\":@ |;<>&`, plus newlines and tabs.
```

Unfortunately, it wouldn't make sense to change this just here, since "left"/"right" are used throughout.

If I wasn't trying to match existing descriptions, I'd spell it out and use what I wrote in https://github.com/fish-shell/fish-shell/issues/2014#issuecomment-945064053 :

> Move to immediately after the last character in the current path component (i.e. right after the previous folder slash if there is one, else to the start of the word).

(Same applies the other new bindings in this PR.)